notification-component-icon/src/NotificationIcon.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {
+  Dropdown,
+  DropdownMenu,
+  DropdownToggle,
+  DropdownItem,
+} from 'reactstrap';
+import FontAwesomeIcon from '@fortawesome/react-fontawesome';
+
+class NotificationIcon extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      dropdownOpen: false,
+      notifications: [
+        {
+          url: 'https://example.com/register',
+          message: 'time to register for finals',
+          isRead: false,
+        },
+        {
+          url: 'https://example.com/game',
+          message: 'get tickets to the big game',
+          isRead: true,
+        },
+      ],
+    };
+  }
+
+  toggle = () => {
+    this.setState({
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  };
+
+  renderEmpty = () => (
+    <DropdownItem className="up-notification--menu-item" disabled>
+      congrats no notifications
```

Comment:
How is i18n normally handled in this tech stack?

Reply:
Usually through [react-intl](https://github.com/yahoo/react-intl)